### PR TITLE
core(pwa): remove load-fast-enough-for-pwa audit

### LINKED ISSUE #10943: Align load-fast-enough-for-pwa with Core Web Vitals?
**Summary**

"Fast enough" is determined by TTI: https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/audits/load-fast-enough-for-pwa.js#L22

Seems like this should be based off of CWV thresholds?

### PR DESCRIPTION
**Pull request by paulirish**
The perf test in PWA has mostly just ended up adding extra confusion.

In a discussion in xfunctional August, we had consensus on removing the requirement from the PWA section. 

Naturally anyone evaluating their PWA can also look at the Perf section. So it's all groovy.

fixes #10943

### CODE REVIEW

**Review by adamraine — APPROVED** [review]
LGTM. I think any necessary changes are covered by CI.

**Review by connorjclark — COMMENTED** [review]
`core(load-fast-enough-for-pwa): remove audit` ?


bye bye

File: lighthouse-core/audits/load-fast-enough-for-pwa.js [hunk 1]
```diff
@@ -1,105 +0,0 @@
...
-  '{timeInMs, number, seconds}\xa0s',
-  /** Explanatory message displayed when a web page loads too slowly to be considered quickly interactive. This references another Lighthouse auditing category, "Performance", that can give additional details on performance debugging.  */
-  explanationLoadSlow: 'Your page loads too slowly and is not interactive within 10 seconds. ' +
-    'Look at the opportunities and diagnostics in the "Performance" section to learn how to ' +
-    'improve.',
-};
-
-const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
-
-class LoadFastEnough4Pwa extends Audit {
```
Comment (brendankenny) [inline lighthouse-core/audits/load-fast-enough-for-pwa.js:44]: wow, the class name made it all the way to the end

**Review by brendankenny — APPROVED** [review]
> `// Ignore speed test; just verify that it ran.`

A fitting send-off summary :)

🎆 🎆 ⚱️ 🎆 🎆

**Pull request merged**